src/components/Select/__tests__/base-actions.test.tsx

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
```

Comment:
Let's name the test file as `Select.test.tsx`, `base-actions` it's just a chunk of tests